pkg/audit/manager.go

```diff
@@ -138,6 +139,36 @@ type updateListEntry struct {
 	enforcementAction util.EnforcementAction
 }
 
+// ByGVKNNMsg implements sort.Interface based on the group, version, kind, name, namespace, and msg fields.
+type byGVKNNMsg []updateListEntry
+
+func (a byGVKNNMsg) Len() int {
+	return len(a)
+}
+
+func (a byGVKNNMsg) Less(i, j int) bool {
+	if a[i].group == a[j].group {
```

Comment:
I think we can remove nesting here by inverting the logic:

```
if a[i].group != a[j].group {
  return a[i].group < a[j].group
}
if a[i].version != a[j].version {

.... and so forth
```

Reply:
Done, thanks!